Raspberry Pi connect gets stuck at `checking status ....`

When Raspberry Pi connect is connected to a session if I click on RPi connect item under remote menu, the next page gets stuck showing `checking status ....`

<img width="230" alt="Screenshot 2024-09-15 at 5 58 06 PM" src="https://github.com/user-attachments/assets/0aef91b7-7778-48cc-a754-d2f5fcb5f3fc">

Software version `0.16.1` lite image.

btw, if I disconnect from the remote session, this page does not show up. In other words, this only happens when there is an open/active remote shell session.

I see the following error in the logs after this happens:

```
1726448087.830788 [ERROR] Event loop exception
Traceback (most recent call last):
  File "/opt/ubo/env/lib/python3.11/site-packages/ubo_app/services/050-rpi-connect/commands.py", line 272, in check_status
    await _check_status()
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 216, in wrapper_
    return await _leading_edge(state, state.last_call_time)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 83, in _leading_edge
    return await _invoke_func(state, time) if state.leading else state.result
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 143, in _invoke_func
    result = await result
             ^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/ubo_app/services/050-rpi-connect/commands.py", line 77, in _check_status
    'remote_shell_sessions': int(data.group(5))
                             ^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType' - extra: {
  "error_message": "Task exception was never retrieved",
  "future": "<Task finished name='Task-14556' coro=<check_status() done, defined at /opt/ubo/env/lib/python3.11/site-packages/ubo_app/services/050-rpi-connect/commands.py:271> exception=TypeError(\"int() argument must be a string, a bytes-like object or a real number, not 'NoneType'\")>",
  "loop": "<_UnixSelectorEventLoop running=True closed=False debug=False>",
  "parent_stack": "None"
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Changelog
 
-## Upcoming
+## Version 1.2.0
 
 - chore: add `log_async_process` to log the output of an async processes in a notification if they don't run successfully
 - refactor(core): housekeeping: rename `extra_information` to `qr_code_generation_instructions` in `ubo_input`, add `.tmpl` extension for extension files, use `textarea` for `LONG` input field type in web dashboard, rename `..._HOST` env variables to `..._ADDRESS`, use underscore thousand separators for big numbers in codebase

